catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/event/impl/SubscriptionImpl.java

```diff
@@ -22,13 +22,15 @@
 import ddf.catalog.event.Subscription;
 
 public class SubscriptionImpl implements Subscription {
-    private Filter filter;
+    protected Filter filter;
```

Comment:
Yeah i reverted the change to make them protected and made them all final nothing should change after construction.